### DISCUSSION TITLE
Jetstream and Publish Subscribe

### DISCUSSION THREAD

**Discussion by alberk8** [nats-io/nats.net#486] [Q&A] [answered]
I have a scenario where data is coming into a Jetstream where consumer will consume and save data to a Database, concurrently is it possible to have a copy available to a browser.

How do I achieve this?  Do I need to initiate a pub/sub in the Jetstream consume foreach?

**Comment by mtmk** [comment, marked as answer]
You can publish to some other topic, that's possible. If you do that you should probably have a different connection to avoid blocking the same TCP connection.
